packages/glimmer-test-helpers/lib/environment.ts

```diff
@@ -800,7 +800,7 @@ export class TestEnvironment extends Environment {
       }
     }
 
-    if ((!isSimple && appendType === 'unknown') || appendType === 'self-get') {
```

Comment:
This to turn "anything with a dot" to possibly a component invocation because closure components. Previously self-get is different but now they actually _do_ have a dot (parts.length > 1) in them.